src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -153,11 +153,32 @@ const KeyboardAwareScrollView = forwardRef<
         const inputHeight = layout.value?.layout.height || 0;
         const point = absoluteY + inputHeight;
 
+        const scrollOutput = (
```

Comment:
Can this function be moved into separate file?

This component already has pretty big amount of lines of code, so let's reduce it 😎 (especially we can do that, since it's pure function)